routes/htmlRoutes.js

```diff
@@ -3,14 +3,26 @@ const db = require("../models");
 module.exports = app => {
   // Load index page
   app.get("/", (req, res) => {
-    db.Example.findAll({}).then(dbExamples => {
+    db.User.findAll({}).then(User => {
```

Comment:
For these routes, we should actually remove the db.X.findAll since we are just displaying the page.